README.md

```diff
@@ -1,2 +1,14 @@
 # acmOpportunities
-Sends AI-curated digests to our club Discord server. 
+The acmOpportunities Discord Bot is a tool designed for students in our club. This bot keeps software engineers and developers informed about the latest job opportunities in the industry by sending job listings directly to the ACM Discord server.
+
+## Key Features
+
+- **Stay Informed:** Keeps up-to-date with the latest SWE job opportunities near Fullerton, CA without having to manually search for them. Sends in up to 10 job opportunities every other day.
+- **Seamless Integration:**  With the integration of PostgreSQL and Discord, acmOpportunities provides a seamless user experience. The bot retrieves job details from the database and delivers them as rich embeds on Discord, ensuring easy access and engagement.
+
+## Example Response
+![image](https://github.com/acmcsufoss/acmOpportunities/assets/116927138/7fb22b06-4b63-4be3-bab5-473095e14fc3)
+
+##
```

Comment:
What is the title of this last README section?

Reply:
For the purposes of creating a horizontal line, use `---`.

Reply:
Note: Use proper heading casing for the heading text in the `README.md`.

Reply:
Resolved: https://github.com/acmcsufoss/acmOpportunities/pull/1/commits/ab61bc4ad59cbcf21b8df00d42f1b8e9a9bbd854